default-config/config

```diff
@@ -285,7 +285,7 @@ AddToMenu   MenuFvwmRoot "Fvwm" Title
 + "&Programs%icons/programs.png%" Popup MenuPrograms
 + "XDG &Menu%icons/apps.png%" Popup XDGMenu
 + "&XTerm%icons/terminal.png%" Exec exec $[infostore.terminal]
-Test (x dmenu_run) + "Run command" Exec exec dmenu_run
+Test (x dmenu_run) + "Run command" Exec exec dmenu_run -nb "#2b4e5e"
```

Comment:
We should move this colour definition to using infostore, and use it here.

Reply:
That's a good suggestion, it would simplify things. Do you suggest only changing the color definition or use infostore for the full command? Perhaps the full command option could be more convenient for those who want to switch to Rofi or something.

Reply:
Sure. Even better. Go for it!